supernode/services/artworkregister/task.go

```diff
@@ -156,25 +162,30 @@ func (task *Task) ConnectTo(_ context.Context, nodeID, sessID string) error {
 		return err
 	}
 
+	var err error
+
 	task.NewAction(func(ctx context.Context) error {
 		node, err := task.pastelNodeByExtKey(ctx, nodeID)
```

Comment:
could you declare node like:
```
var node *Node
node, err = ...
```
to prevent confusing with local scope

Reply:
updated. Thanks.